nens_auth_client/templates/nens_auth_client/invitation.html

```diff
@@ -0,0 +1,5 @@
+<p>Dear madam, sir,</p>
+<p>You have been invited to access {{ host }}.<br />Use the following link to accept this invitation:</p>
```

Comment:
"access" is ook wat raar naar m'n smaak.

Hoewel... je hebt het waarschijnlijk van lizard-auth-server gekopieerd?